yjit/src/codegen.rs

```diff
@@ -5683,6 +5710,40 @@ fn gen_send_iseq(
         argc = lead_num;
     }
 
+    if iseq_has_rest {
+        assert!(argc >= required_num);
+
+        jit_save_pc(jit, asm);
+        gen_save_sp(jit, asm, ctx);
+
+        // We are going to allocate so setting pc and sp.
+        let n = (argc - required_num) as u32;
+        argc = required_num + 1;
+        // If n is 0, then elts is never going to be read, so we can just pass null
+        let values_ptr = if n == 0 {
+            Opnd::UImm(0)
+        } else {
+            asm.comment("load pointer to array elts");
+            let offset_magnitude = (SIZEOF_VALUE as u32) * n;
```

Comment:
```suggestion
            let offset_magnitude = SIZEOF_VALUE_I32 * n;
```

Reply:
Reverting this. This is a u32 not an i32.

Reply:
I'm sure we could make this code better on the conversion. But it is literally copied and pasted from elsewhere. It would probably better to just make a PR doing all this code clean up for how people want these things to be.

Reply:
Oops, my bad.